frontend/lib/evictionfree/components/background-image.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { DetailedHTMLProps, ImgHTMLAttributes } from "react";
+import { AppContext } from "../../app-context";
+
+type ImgProps = DetailedHTMLProps<
+  ImgHTMLAttributes<HTMLImageElement>,
+  HTMLImageElement
+>;
+
+export type StaticImageProps = ImgProps & {
+  src: string;
+  alt: string;
+};
+
+export function BackgroundImage(props: StaticImageProps): JSX.Element {
+  const { ...imgProps } = props;
+  return (
+    <AppContext.Consumer>
+      {(appCtx) => (
```

Comment:
I assume this has been copypasta'd from somewhere, but this would be more readable as just `const appCtx = useContext(AppContext);` in the function.  NOT A BIG DEAL AT ALL THO.

Reply:
done!